tf2_ros/include/tf2_ros/buffer_interface.h

```diff
@@ -55,11 +55,10 @@ namespace tf2_ros
     return time_msg;
   }
 
-  inline const tf2::TimePoint& fromMsg(const builtin_interfaces::msg::Time & time_msg)
+  inline const tf2::TimePoint fromMsg(const builtin_interfaces::msg::Time & time_msg)
```

Comment:
It depends on what you want, but right now in line 61 you return a non-const object, so the const-return has no effect.